src/components/WebexActivityStream/WebexActivityStream.js

```diff
@@ -157,9 +158,4 @@ export default function WebexActivityStream(props) {
 
 WebexActivityStream.propTypes = {
   roomID: PropTypes.string.isRequired,
```

Comment:
Remove trailing comma, please
```suggestion
  roomID: PropTypes.string.isRequired
```